### ISSUE TITLE
bug when using la with register addressing

### ISSUE THREAD

**Issue by XavierCooney** [insou22/mipsy#286]
The following program prints `32767`, as would be expected:
```
main:
	li	$a0, 0
	la	$a0, 32767($a0)
	li	$v0, 1
	syscall

	li	$v0, 0
	jr	$ra
```
but this one prints `65536` (double the expected answer):
```
main:
	li	$a0, 0
	la	$a0, 32768($a0)
	li	$v0, 1
	syscall

	li	$v0, 0
	jr	$ra
```
Probably something similar going on to #283, since this only seems to occur when the two registers in the `la` instruction are the same.

**Comment by Dylan-Brotherston** [comment]
Assuming the first example is supposed to use something like $t0 instead of $a0

Looks like there is a typo here
https://github.com/insou22/mipsy/blob/030526ec5200cdece7293c9e56f22e9edd18eadc/mips.yaml#L2457
And $Rs is used twice instead of $Rt

**Comment by Dylan-Brotherston** [comment]
That seems to have done the job
![image](https://github.com/insou22/mipsy/assets/22044664/8ee739a0-56a4-4279-af5b-3ee6fea8fe81)

```diff
diff --git a/mips.yaml b/mips.yaml
index aa4f815..a1c719b 100644
--- a/mips.yaml
+++ b/mips.yaml
@@ -2441,11 +2441,11 @@ pseudoinstructions:
       format: [Rs, Off32Rt]
     expand:
       - inst: LUI
-        data: [$Rs, $Off32uHi]
+        data: [$At, $Off32uHi]
       - inst: ORI
-        data: [$Rs, $Rs, $Off32uLo]
+        data: [$At, $At, $Off32uLo]
       - inst: ADDU
-        data: [$Rs, $Rs, $Rt]
+        data: [$Rs, $Rs, $At]

   - name: NOP
     desc_short: No-Operation - doesn't do anything
```

**Comment by Dylan-Brotherston** [comment]
Fixed by #288
